browser-tests/functional-tests/accessibility.spec.js

```diff
@@ -0,0 +1,40 @@
+const { test, expect } = require("@playwright/test");
+const { iteratePagesAndContexts } = require("../data/pagesAndContexts.js");
+const AxeBuilder = require("@axe-core/playwright");
+
+test.describe.parallel("Accessibility tests", () => {
+  test.setTimeout(120000);
+  iteratePagesAndContexts(
+    (pageName, context, language, url) => {
+      test(`Accessibility check for ${pageName}, context ${context} and language ${language}`, async ({ page }) => {
+        await page.goto(url);
+        await assertNoAccessibilityViolations(page);
+      });
+    }
+  );
+
+  (["radio", "checkbox"]).forEach((errorState) => {
+    ["en", "cy"].forEach((language) => {
+      test(`Accessibility check for ${errorState} error state in ${language}`, async ({ page }) => {
+        await page.goto(`http://localhost:4601/dev/template/confirm-your-details/${language}?errorState=${errorState}`);
+        await assertNoAccessibilityViolations(page);
+      });
+    })
+  });
+
+  ["en", "cy"].forEach((language) => {
+    test(`Accessibility check for open details in ${language}`, async ({ page }) => {
+      await page.goto(`http://localhost:4601/dev/template/pyi-triage-select-smartphone/${language}`);
+      await page.click(".govuk-details__summary");
```

Comment:
The docs mention something about using `.waitFor()` - do we need to do that?
>   // It is important to waitFor() the page to be in the desired
  // state *before* running analyze(). Otherwise, axe might not
  // find all the elements your test expects it to scan.

Reply:
The tests passed without this but I will include just in case!